addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -13,13 +13,12 @@
   },
   "private": true,
   "dependencies": {
-    "angular2": "2.0.0-beta.9",
-    "es6-promise": "^3.0.2",
```

Comment:
I think that's a bug in Angular. With the new zones update we can now drop
the es6-promise polyfill
On Fri, Mar 18, 2016 at 7:20 PM Filipe Silva <notifications@github.com>
wrote:

> In addon/ng2/blueprints/ng2/files/package.json
> <https://github.com/angular/angular-cli/pull/321#discussion_r56742311>:
>
> > @@ -13,13 +13,12 @@
> >    },
> >    "private": true,
> >    "dependencies": {
> > -    "angular2": "2.0.0-beta.9",
> > -    "es6-promise": "^3.0.2",
>
> Was es6-promise rolled into some other polyfill? I still see it in
> https://github.com/angular/angular/blob/master/package.json#L35
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/angular-cli/pull/321/files/ed8c80802cbc0c7edd54e3f433789ed8f3ff8b5a#r56742311>
>